bitcoinetl/cli/stream.py

```diff
@@ -41,6 +41,7 @@
 @click.option('-o', '--output', type=str,
               help='Google PubSub topic path e.g. projects/your-project/topics/bitcoin_blockchain. '
                    'If not specified will print to console.')
+@click.option('--topic-mapping', default=None, type=dict[str,str], help="Topic Mapping should be Python dict like {'block': 'producer.bitcoin.hot.blocks','transaction': 'producer.bitcoin.hot.transactions',}")
```

Comment:
topic names have changed
```suggestion
@click.option('--topic-mapping', default=None, type=dict[str,str], help="Topic Mapping should be Python dict like {'block': 'producer-bitcoin-blocks-hot','transaction': 'producer-bitcoin-transactions-hot'}")
```